src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Model/CBORWebTokenHeader.swift

```diff
@@ -0,0 +1,22 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+public struct CBORWebTokenHeader: Codable, Equatable {
+
+    // MARK: - Protocol Codable
+
+    enum CodingKeys: String, CodingKey {
+        case issuer = "iss"
+        case issuedAt = "iat"
+        case expirationTime = "exp"
+    }
+
+    // MARK: - Internal
+
+    let issuer: String
+    let issuedAt: UInt64?
+    let expirationTime: UInt64
```

Comment:
Could these be `Date`s? and be named `issueDate` and `expirationDate`? But we could also convert that on the Service level / introduce computed properties for that I guess 🤔